### DISCUSSION TITLE
MCDU Menu Button

### DISCUSSION THREAD

**Discussion by Whiskey68** [flybywiresim/aircraft#2306] [Help] [answered]
When I press the MCDU button, nothing happens. In the tutorial, I am watching that's how you access things I need for planning like ATIS. Is this just on my end or a general bug?

**Comment by ImenesFBW** [comment, marked as answer]
You need to use the development version for those options.

**Reply by Whiskey68** [reply]
Great! Thank you for the clarification. Ill DL it now!
